src/index.js

```diff
@@ -1,2 +1,2 @@
-export Notification from './notification';
-export NotificationStack from './notificationStack';
+export * from './notification';
+export * from './notificationStack';
```

Comment:
Since both of these modules have no named exports, these two will not export anything.

Please change this to the form
`export { default as Notification } from './notification';`